test/Makefile

```diff
@@ -0,0 +1,28 @@
+# http://clarkgrubb.com/makefile-style-guide
+
+MAKEFLAGS += --warn-undefined-variables --no-print-directory
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := test
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+tag = claranet/direnv-asdf
+
+.PHONY: test
+test:
+	mkdir -p .direnv .docker/.direnv .docker/home
+	docker run --rm \
+		-v $(PWD):/src \
+		-v $(PWD)/.docker/.direnv:/src/.direnv \
+		-v $(PWD)/.docker/home:$(HOME) \
```

Comment:
Mount these paths into a `.docker` directory so we get caching on the host without polluting the main directory on the host.